app/services/DateService.scala

```diff
@@ -17,19 +17,16 @@
 package services
 
 import config.FrontendAppConfig
-import config.featureswitch.{FeatureSwitching, TimeMachineAddYear}
 
 import java.time.LocalDate
 import java.time.Month.APRIL
 import javax.inject.{Inject, Singleton}
 
 @Singleton
-class DateService @Inject()(implicit val frontendAppConfig: FrontendAppConfig) extends DateServiceInterface with FeatureSwitching {
+class DateService @Inject()(implicit val frontendAppConfig: FrontendAppConfig) extends DateServiceInterface{
 
-  override lazy val appConfig: FrontendAppConfig = implicitly
-
-  def getCurrentDate: LocalDate = {
-    if (isEnabled(TimeMachineAddYear)) {
+  def getCurrentDate(isTimeMachineEnabled: Boolean): LocalDate = {
```

Comment:
we can reduce our changes a bit by setting default to false, as this would be a primary usage

Reply:
Have done now, will look to see where I am calling with just (false) and remove that